server/election/leadership.go

```diff
@@ -34,7 +34,7 @@ func GetLeader(c *clientv3.Client, leaderPath string) (*pdpb.Member, int64, erro
 	leader := &pdpb.Member{}
 	ok, rev, err := etcdutil.GetProtoMsgWithModRev(c, leaderPath, leader)
 	if err != nil {
-		return nil, 0, err
+		return nil, 0, errs.ErrEtcdGetProtoMsgWithModRev.Wrap(err).FastGenWithCause()
```

Comment:
i think we should handle errors in the lower level

Reply:
You're right.

Reply:
updated